recipes/default.rb

```diff
@@ -1,6 +1,12 @@
 if platform?('ubuntu') && node['platform_version'] >= '18.04'
+  replace_or_add 'Fix systemd-resolved conflict' do
+    path '/etc/systemd/resolved.conf'
+    pattern 'DNSStubListener=*'
+    line 'DNSStubListener=no'
+  end
+
   service 'systemd-resolved' do
-    action :stop
+    action :restart
```

Comment:
As written, this will cause `systemd-resolved` to restart every invocation of `chef-client`

Reply:
you are 100% correct, i will fix it now and push a new commit.